### ISSUE TITLE
OAS : Semantic errors in the "responses" block of the operationid "getStringProfileStringValue"

### ISSUE THREAD

**Issue by PrathibaJee** [openBackhaul/OperationKeyManagement#1]
Observed the following semantic errors in the response block of the operationId "getStringProfileStringValue"

```
Semantic error at paths./core-model-1-4:control-construct/profile-collection/profile=okm-0-0-1-string-p-0000/string-profile-1-0:string-profile-pac/string-profile-configuration/string-value.put.responses.400.$ref
$refs must reference a valid location in the document
Jump to line 7033
Semantic error at paths./core-model-1-4:control-construct/profile-collection/profile=okm-0-0-1-string-p-0000/string-profile-1-0:string-profile-pac/string-profile-configuration/string-value.put.responses.401.$ref
$refs must reference a valid location in the document
Jump to line 7035
Semantic error at paths./core-model-1-4:control-construct/profile-collection/profile=okm-0-0-1-string-p-0000/string-profile-1-0:string-profile-pac/string-profile-configuration/string-value.put.responses.403.$ref
$refs must reference a valid location in the document
Jump to line 7037
Semantic error at paths./core-model-1-4:control-construct/profile-collection/profile=okm-0-0-1-string-p-0000/string-profile-1-0:string-profile-pac/string-profile-configuration/string-value.put.responses.404.$ref
$refs must reference a valid location in the document
Jump to line 7039
Semantic error at paths./core-model-1-4:control-construct/profile-collection/profile=okm-0-0-1-string-p-0000/string-profile-1-0:string-profile-pac/string-profile-configuration/string-value.put.responses.500.$ref
$refs must reference a valid location in the document
Jump to line 7041
Semantic error at paths./core-model-1-4:control-construct/profile-collection/profile=okm-0-0-1-string-p-0000/string-profile-1-0:string-profile-pac/string-profile-configuration/string-value.put.responses.default.$ref
$refs must reference a valid location in the document
Jump to line 7043
```
Need to change the reference in the response code to `'#/components/responses/responseForErroredOamRequests'`

**Comment by MartinSunal** [comment]
this has been fixed in #7

**Comment by openBackhaul** [comment]
StringProfile definition is part of ApplicationPattern.
In the ApplicationPattern OAS, no differences can be found in the responses > 204 compared with other OAM requests.
Already done.
